### ISSUE TITLE
As a content consumer, I need the ability to turn closed captions on and off. 

### ISSUE THREAD

**Issue by simcaleer** [carbon-design-system/ibm-dotcom-library#4012]
### The problem
IBM is dedicated to accessibility and making an inclusive experience available to content consumers. In the default video player, the closed caption button does not seem to be the default. That option though is apparently available in the Kaltura video player. We should make the closed caption button the default for the ibm.com video player?

Out of the 165 survey respondents, 162 (98%) said they watch videos with the closed captions turned on. https://www.3playmedia.com/2017/04/21/infographic-closed-captions-use-survey/

### The solution
When developers embed a video player on their ibm.com site, they would tend to use the default listed within the Carbon PAL. This default should have the closed caption button enabled by default. This will force folks to intentionally create the captions or turn off the button. 

### Additional information
This is where I found the option in the Kaltura video player. http://player.kaltura.com/docs/AccessibilityControls
This is where I found our own template for a video player https://www.ibm.com/standards/web/ibm-dotcom-library/components/video-player/

Out of the 165 survey respondents, 162 (98%) said they watch videos with the closed captions turned on. https://www.3playmedia.com/2017/04/21/infographic-closed-captions-use-survey/
